en/docs/learn/examples/inbound-endpoint-examples/inbound-endpoint-hl7-protocol-auto-ack.md

```diff
@@ -24,13 +24,10 @@ Following are the integration artifacts that we can used to implement this scena
     ```xml
     <?xml version="1.0" encoding="UTF-8"?>
     <sequence name="main" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
```

Comment:
Let's rename the sequence. main sequence is a special sequence we used, but now that is removed. So, it is better to use a different name for this one